source/triggers/aws-eventbridge.txt

```diff
@@ -96,22 +101,29 @@ Procedure
 
             Set the ``account_id`` field to the :guilabel:`AWS Account ID`
             that you copied from EventBridge and set the ``region`` field to
-            an AWS Region.
+            an AWS Region. 
+            
+            Optionally, you can serialize event objects into 
+            :manual:`Extended JSON format </reference/mongodb-extended-json>`. 
+            To enable Extended JSON, set the ``extended_json_enabled`` 
+            field to ``true``.
```

Comment:
Can we state the default here? I'm assuming it's `extended_json_enabled: false` but not sure.

Reply:
e.g. "By default, triggers convert the BSON types in event objects into standard JSON types. To preserve type information, you can optionally serialize event objects into Extended JSON format instead."